.travis.yml

```diff
@@ -14,56 +13,24 @@ addons:
   #     - postgresql-9.6-postgis-2.3
 
 python:
-  - "2.7"
-  - "3.4"
-  - "3.5"
   - "3.6"
   - "3.7"
+  - "3.8"
 
 env:
-  - DJANGO_VERSION='Django>=1.8,<1.9'
-  - DJANGO_VERSION='Django>=1.9,<1.10'
-  - DJANGO_VERSION='Django>=1.10,<1.11'
-  - DJANGO_VERSION='Django>=1.11,<2.0'
-  - DJANGO_VERSION='Django>=2.0,<2.1'
-  - DJANGO_VERSION='Django>=2.1,<2.2'
+  - DJANGO_VERSION='Django>=1.11,<1.12'
+  - DJANGO_VERSION='Django>=2.2,<2.3'
+  - DJANGO_VERSION='Django>=3.0,<3.1'
   - DJANGO_VERSION='https://github.com/django/django/archive/master.tar.gz'
 
-matrix:
-  exclude:
-    - python: "2.7"
-      env: DJANGO_VERSION='Django>=2.0,<2.1'
-    - python: "2.7"
-      env: DJANGO_VERSION='Django>=2.1,<2.2'
-    - python: "3.4"
-      env: DJANGO_VERSION='Django>=2.1,<2.2'
-    - python: "2.7"
-      env: DJANGO_VERSION='https://github.com/django/django/archive/master.tar.gz'
-    - python: "3.4"
-      env: DJANGO_VERSION='https://github.com/django/django/archive/master.tar.gz'
-  allow_failures:
-    - python: "3.6"
-      env: DJANGO_VERSION='Django>=1.8,<1.9'
-    - python: "3.7"
-      env: DJANGO_VERSION='Django>=1.8,<1.9'
-    - python: "3.6"
-      env: DJANGO_VERSION='Django>=1.9,<1.10'
-    - python: "3.7"
-      env: DJANGO_VERSION='Django>=1.9,<1.10'
-    - python: "3.6"
-      env: DJANGO_VERSION='Django>=1.10,<1.11'
-    - python: "3.7"
-      env: DJANGO_VERSION='Django>=1.10,<1.11'
-    - env: DJANGO_VERSION='https://github.com/django/django/archive/master.tar.gz'
```

Comment:
We should definitely allow Django `master` to fail without failing the build. There's been plenty of times where PRs for django-cities fail due to incompatible changes in upstream Django.